### PR TITLE
Add the documents needed to contribute

### LINKED ISSUE #3: Add a CONTRIBUTING.md
https://docs.github.com/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Simplified version of [github/docs](https://github.com/github/docs/blob/main/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by blendthink**
## Issue

- close #3 

## Overview (Required)

Add the following documents needed to contribute:

- CONTRIBUTING.md
- SELF_REVIEW.md

### CODE REVIEW

File: docs/contributing/SELF_REVIEW.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Self-Review
+
+Before submitting your Pull Request, please check the following items:
+
+- [ ] Are there any typos?
```
Comment (YumNumm) [inline docs/contributing/SELF_REVIEW.md:5]: imo-badge
typoが無い時に `[x]` とし、全てチェック済みになるほうが分かりやすいような気がしました。
```suggestion
- [ ] Are there no typos?
```

File: docs/contributing/SELF_REVIEW.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- [ ] Are all the links included?
+- [ ] Have you made any unnecessary changes?
```
Comment (YumNumm) [inline docs/contributing/SELF_REVIEW.md:7]:
```suggestion
- [ ] Have you made no unnecessary changes?
```

File: docs/contributing/SELF_REVIEW.md (hunk 1 above)
Comment (blendthink) [reply docs/contributing/SELF_REVIEW.md]: Fixed in the following commit:
https://github.com/yumemi-inc/flutter-yumemi-lints/pull/40/commits/da1c108acf0523ee4b0970fcbfa5e1737972267c

File: docs/contributing/SELF_REVIEW.md (hunk 2 above)
Comment (blendthink) [reply docs/contributing/SELF_REVIEW.md]: Fixed in the following commit:
https://github.com/yumemi-inc/flutter-yumemi-lints/pull/40/commits/da1c108acf0523ee4b0970fcbfa5e1737972267c

**Review by trm11tkr — COMMENTED** [review]
Documentation looks good. Thanks for the addition! I have one question, though.

I've seen some projects include a `CODE_OF_CONDUCT.md` and link to it from their `CONTRIBUTING.md`. Given the smaller scale of this project, have we chosen not to include one?

FYI:
- [flutter](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md#:~:text=our%20community%20norms%3A-,Our%20code%20of%20conduct%2C,-which%20stipulates%20explicitly)
- [riverpod](https://github.com/rrousselGit/riverpod/blob/master/CONTRIBUTING.md#:~:text=check%20out%20our-,Code%20of%20conduct.,-You%20will%20also)

**Comment by blendthink** [comment]
@trm11tkr 
> Given the smaller scale of this project, have we chosen not to include one?

Yes!

**Review by trm11tkr — APPROVED** [review]
LGTM!

**Pull request merged**